portfolio/src/main/webapp/index.html

```diff
@@ -73,6 +67,25 @@ <h1 id="greeting-container"></h1>
           </a>
         </div>
       </div>
+      <br>
```

Comment:
Tiny nit: it's probably better to leave out `<br>` tags like these and instead use CSS properties like `margin-top` or `margin-bottom` to get blank space

Reply:
Will do.